types/errors.go

```diff
@@ -7,28 +7,30 @@ import (
 	"github.com/tendermint/go-crypto"
 )
 
+// ABCI Response Code
 type CodeType uint32
 
+// is everything okay?
 func (code CodeType) IsOK() bool {
 	if code == CodeOK {
 		return true
-	} else {
-		return false
 	}
+	return false
 }
 
+// ABCI Response Codes
+// Base SDK reserves 0 ~ 99.
 const (
-	// ABCI Response Codes
-	// Base SDK reserves 0 ~ 99.
 	CodeOK                  CodeType = 0
 	CodeInternal            CodeType = 1
 	CodeTxParse             CodeType = 2
-	CodeBadNonce            CodeType = 3
-	CodeUnauthorized        CodeType = 4
-	CodeInsufficientFunds   CodeType = 5
-	CodeUnknownRequest      CodeType = 6
-	CodeUnrecognizedAddress CodeType = 7
-	CodeInvalidSequence     CodeType = 8
+	CodeGenesisParse        CodeType = 3
```

Comment:
weird seeing this in here. InitChain doesn't even return a code - do we need this ?

Reply:
SetInitStater can return error when initializing genesis - what error code would you use otherwise? CodeInternal?